targets/csharp-unity/source/Internal/ISerializer.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace  PlayFab
+{
+    public interface ISerializer
+    {
+        T DeserializeObject<T>(string json);
+        T DeserializeObject<T>(string json, object jsonSerializerStrategy);
+
+        string SerializeObject(object json);
+        string SerializeObject(object json, object jsonSerializerStrategy);
+    }
+
+    public class SimpleJson
+    {
+        private static ISerializer _instance = new SimpleJsonInstance();
+
+        public static ISerializer Instance
```

Comment:
This needs a docstring to indicate that this is the global reset for changing out the serializer.